mqtt3/1.0.0/modules/ROOT/pages/mqtt3-documentation.adoc

```diff
@@ -0,0 +1,490 @@
+= MQTT3 Connector 1.0 Reference - Mule 4
```

Comment:
Why is the filename of the reference mqtt3-documentation.adoc instead of mqtt3-reference.adoc?